rage/examples/generate-docs.rs

```diff
@@ -98,35 +98,14 @@ fn rage_page() {
             Example::new()
                 .text("Encryption to a list of recipients at an HTTPS URL")
                 .command(
-                    "echo \"_o/\" | rage -o hello.age \
-                     -r https://github.com/str4d.keys > hello.age",
+                    "echo \"_o/\" | rage -o hello.age -r <(curl https://github.com/str4d.keys)",
```

Comment:
This being equivalent, but almost as succinct, is part of why I'm happy to remove HTTPS support from `rage` itself. BY comparision, trying to convert a file containing a list of recipients into a sequence of `-r` flags is _much_ more complex.